packages/fiori/src/styles/ToolLayout.module.css

```diff
@@ -0,0 +1,36 @@
+.shellBar::part(root) {
+    padding-inline-start: 0.75rem;
+    padding-inline-end: 1.25rem;
+}
+
```

Comment:
@alexandar-mitsev could you check if this is abstract enough as discussed offline?